domains/farrellid.is-a.dev.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "fidxteam",
+    "email": "dev.farrellid@gmail.com"
+  },
+  "record": {
+    "CNAME": "cname.vercel-dns.com",
+    "TXT": {
+      "_vercel": "vc-domain-verify=farrellid.is-a.dev,ab8acadd131fd3db1c7f"
+  }
```

read the step 5 -> https://docs.is-a.dev/guides/vercel/
```suggestion
    "CNAME": "cname.vercel-dns.com"
```